Documentation/UsingSetting/Constants.rst

```diff
@@ -7,14 +7,30 @@
 Constants
 =========
 
-..  include:: /Images/AutomaticScreenshots/TemplatesModul/TemplatesConstants.rst.txt
+..  versionchanged:: 12.0
+    The TypoScript management tools are now found in backend module
+    :guilabel:`Site Management > TypoScript`. It was formerly found in
+    "Page > Template".
+
+..  figure:: /Images/ManualScreenshots/TypoScriptModule/ConstantEditor.png
+    :alt: Screenshot of the TYPO3 Backend showing the constant editor
+
+    The :ref:`Constant Editor <constant-editor>` in the TYPO3 backend
 
 Constants are values defined in the  :guilabel:`Constants` field of a template.  They
 follow the :ref:`syntax of ordinary TypoScript
 <t3coreapi:typoscript-syntax-syntax>` and are case sensitive! They are used to
 manage *in a single place* values, which are later used in *several places*.
 
+..  seealso::
+    Most constants can be edited in the :guilabel:`Template` module using the
```

Comment:
How about "assigned" instead of "edited"? We don't really edit the definitions, but apply values to defined constants...? 

(I'm not too firm on this section so please excuse me if I misunderstand)

Reply:
Done